### ISSUE TITLE
{game_name} not resolving for vod filename or folder

### ISSUE THREAD

**Issue by olig89** [MrBrax/LiveStreamDVR#470]
Files and folders are being created with everything resolving apart form {game_name}.

This is occurring both for vod downloads (from the menu), and post-stream downloads (triggered automatically) - both occasions where game/category data should be available.

**Comment by MrBrax** [comment]
video info fetched from the twitch api unfortunately do not have game/chapter information attached

**Comment by MrBrax** [comment]
some rather unorthodox ways of getting this info has now been implemented onto the dev branch

**Comment by olig89** [comment]
Cool! Look forward to testing it, will have to sub to some randoms as my main (usecase) streamer is offline for a bit.

**Comment by olig89** [comment]
Looks like your wizardry works! I'm getting the game name resolved for both file and folder fields now.

My next issue is with using that field to create a subfolder but that's a separate problem, so I'll create a different issue to keep this one tidy.

**Comment by olig89** [comment]
Looks like this has reverted back to not resolving (on develop branch)

**Comment by MrBrax** [comment]
yes there's a new option about enabling gql under twitch, it is an unsupported and non encouraged feature

**Comment by olig89** [comment]
Quite a weird one, on latest dev build I have `Cities: Skylines II` behaving in a way i've not seen anything do before.

For other games it's fine but for this one it will show as this in putty:
![image](https://github.com/MrBrax/LiveStreamDVR/assets/24813487/a4907a21-fc01-4dea-97db-cf04ef4ff4ba)

but from windows, like this: 
![image](https://github.com/MrBrax/LiveStreamDVR/assets/24813487/c2747714-51b8-4b95-ad04-597c26591751)

I wouldn't care but because It's a weird set of characters, LivestreamDVR then makes a new folder for each new video (with similarly random string).

Within that folder there are the season folders, and files, as expected - only this game folder that's misbehaving

**Comment by MrBrax** [comment]
classic case of smb not being able to handle weird filenames yes

**Comment by MrBrax** [comment]
newest dev version should hopefully fix this now, but it also includes some major breaking changes that might wipe vod json files, not sure yet. make backups.
